<?xml version="1.0" encoding="UTF-8"?>
<ns2:test-suite xmlns:ns2="urn:model.allure.qatools.yandex.ru" start="1459657786297" stop="1459657816335">
	<name>Login page...</name>
	<title>Login page...</title>
	<test-cases>
		<test-case start="1459657786298" stop="1459657816334" status="failed">
			<name>should allow users to login with valid credentials</name>
			<title>should allow users to login with valid credentials</title>
			<description></description>
			<labels>
			</labels>
			<steps>
			</steps>
			<attachments>
			</attachments>
			<failure>
				<message>Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL.</message>
				<stack-trace>Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL.
    at [object Object]._onTimeout (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:1812:23)
    at Timer.listOnTimeout (timers.js:93:15)</stack-trace>
			</failure>
		</test-case>
	</test-cases>
</ns2:test-suite>